apps/website/src/app/docs/rating-icon/page.tsx

```diff
@@ -0,0 +1,106 @@
+"use client";
+
+import { Rating } from "@namehash/nameguard";
+import {
+  RatingIcon,
+  RatingIconSize,
+  RatingLoadingIcon,
+} from "@namehash/nameguard-react";
+
+export default function RatingShieldsPage() {
+  return (
+    <div className="py-12 max-w-7xl mx-auto px-6 space-y-12">
+      <div className="divide-y divide-gray-100">
+        <h1 className="justify-center flex font-bold text-2xl mb-4">
+          {"<"}RatingIcon {"/>"} documentation
+        </h1>
+
+        <div className="grid grid-cols-5 gap-x-6 py-5 text-center font-medium mt-12">
+          <div></div>
+          <div>Rating.pass</div>
+          <div>Rating.warn</div>
+          <div>Rating.alert</div>
+          <div>Loading Rating</div>
```

Comment:
What about Unknown? Suggest we create a distinct table for these as they are a distinct UI component (not a `RatingIcon`).

Also, there should be a distinct table for these too because they also are not `RatingIcon` from a technical perspective.

![CleanShot 2024-05-07 at 17 01 39](https://github.com/namehash/nameguard/assets/126201998/2ca7cc10-7e44-4091-82dd-254a5e949a95)

Reply:
Hi @lightwalker-eth !

I've defined the following tables:

`RatingIcon` table (`/docs/rating-icon`):

![Captura de Tela 2024-05-08 às 17 35 47](https://github.com/namehash/nameguard/assets/49823133/ba098b47-4b37-42d4-aabb-704397124e72)


`ReportIcon` table (`/docs/report`):

![Captura de Tela 2024-05-08 às 17 34 47](https://github.com/namehash/nameguard/assets/49823133/68dff942-f5d0-4f40-8599-ad7462e5005f)